packages/rust_transcription/benchmark.py

```
@@ -22,6 +23,13 @@
 GPU_PORT = 7772
 
 
+def assert_http_url(url: str) -> None:
+    """Reject non-http(s) URLs before handing them to urllib."""
+    scheme = urlparse(url).scheme.lower()
+    if scheme not in ("http", "https"):
+        raise ValueError(f"Refusing to open non-http(s) URL: {url}")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)